docs/plugins/create.md

```diff
@@ -136,6 +136,19 @@ See also the related hooks:
 - `post_asdf_plugin_add`
 - `post_asdf_plugin_add_${plugin_name}`
 
+#### bin/post-plugin-update
+
+This can be used to run any post-plugin-update actions after the plugin has been updated to asdf.
```

Comment:
I'm not sure the `... to asdf.` ending adds anything meaningful here. I think `This can be used to run any post-plugin-update actions after asdf has downloaded the updated plugin` might be better wording here.